bin/avc-test-attack

```diff
@@ -70,20 +71,31 @@ def test_attack(directory, no_cache, gpu, mode, samples):
     print('Submission folder: "{}"'.format(directory))
 
     # build image
-    if no_cache:
-        print('Building image (without cache)...')
-        raise AssertionError('Option does not yet exist because repo2docker'
-                             ' does not allow us to build without cache.')
-        subprocess.check_call(
-            "crowdai-repo2docker --no-run --image-name {} --debug {}".format(
-                image_name, directory),
-            shell=True)
-    else:
-        print('Building image from cache (if exists)...', end="")
+    repo2docker_args = ["--no-run", "--debug"]
+    if not no_build:
```

Comment:
Just some nitpicking: I'd rather switch the if/else-block using `if no_build:` or name the variable `build` only and then have `if build:`... I find the double negation `it not no_build:` to be a bit irritating...